snap/squashfs/squashfs.go

```diff
@@ -72,14 +72,15 @@ func (s *Snap) Install(targetPath, mountDir string) error {
 	// it to the location which is good enough for the tests.
 	if osutil.GetenvBool("SNAPPY_SQUASHFS_UNPACK_FOR_TESTS") {
 		if err := s.Unpack("*", mountDir); err != nil {
-			return err
+			return false, err
 		}
 	}
 
 	// nothing to do, happens on e.g. first-boot when we already
 	// booted with the OS snap but its also in the seed.yaml
 	if s.path == targetPath || osutil.FilesAreEqual(s.path, targetPath) {
-		return nil
+		didNothing := true
+		return didNothing, nil
```

Comment:
why not just

```suggestion
		return true, nil
```

Reply:
I don't have strong opinion on this, but it was deliberate, so that the intent and meaning of 'true' is clear and easy to spot.

Reply:
got it, that's fine thanks for clarifying